### PR TITLE
introduce tektoninstallation CRD

### PR DESCRIPTION
**Pull request by dipak-pawar**

### CODE REVIEW

File: deploy/crds/toolchain.openshift.dev_v1alpha1_tektoninstallation_cr.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: toolchain.openshift.dev/v1alpha1
+kind: TektonInstallation
+metadata:
+  name: example-tektoninstallation
+spec:
+  # Add fields here
+  size: 3
```
Comment (alexeykazakov) [inline deploy/crds/toolchain.openshift.dev_v1alpha1_tektoninstallation_cr.yaml]: Spec is empty for now.

File: deploy/crds/toolchain.openshift.dev_tektoninstallations_crd.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+metadata:
+  name: tektoninstallations.toolchain.openshift.dev
+spec:
+  group: toolchain.openshift.dev
+  names:
+    kind: TektonInstallation
+    listKind: TektonInstallationList
+    plural: tektoninstallations
+    singular: tektoninstallation
+  scope: Namespaced
```
Comment (alexeykazakov) [inline deploy/crds/toolchain.openshift.dev_tektoninstallations_crd.yaml]: And btw, I think we should make it cluster scoped. And the same for `CheInstallation`.

Comment (dipak-pawar) [reply deploy/crds/toolchain.openshift.dev_tektoninstallations_crd.yaml]: Addressed 942585f

File: deploy/crds/toolchain.openshift.dev_v1alpha1_tektoninstallation_cr.yaml (hunk 1 above)
Comment (dipak-pawar) [reply deploy/crds/toolchain.openshift.dev_v1alpha1_tektoninstallation_cr.yaml]: addressed in 942585f

**Pull request merged**